grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -1,82 +1,113 @@
+  const onSubmit = async (data: FormData) => {
+    const transformedData = prepareForUpdate(selectedUserResponders, selectedTeamResponder, data);
 
-      const resp = await directPagingStore.createManualAlertRule(transformedData);
+    const resp = await directPagingStore.createManualAlertRule(transformedData);
 
-      if (!resp) {
-        openWarningNotification('There was an issue creating the alert group, please try again');
-        return;
-      }
+    if (!resp) {
+      openWarningNotification('There was an issue creating the alert group, please try again');
+      return;
+    }
 
-      directPagingStore.resetSelectedUsers();
-      directPagingStore.resetSelectedTeam();
+    directPagingStore.resetSelectedUsers();
+    directPagingStore.resetSelectedTeam();
 
-      onCreate(resp.alert_group_id);
-      onHide();
-    },
-    [prepareForUpdate, selectedUserResponders, selectedTeamResponder]
-  );
+    onCreate(resp.alert_group_id);
+    onHide();
+  };
+
+  const styles = useStyles2(getStyles);
 
   return (
     <Drawer scrollableContent title="New escalation" onClose={onHideDrawer} closeOnMaskClick={false} width="70%">
       <VerticalGroup>
-        <GForm form={manualAlertFormConfig} data={data} onSubmit={handleFormSubmit} onChange={setFormIsValid} />
-        <AddResponders mode="create" />
-        <div className="buttons">
-          <HorizontalGroup justify="flex-end">
-            <Button variant="secondary" onClick={onHideDrawer}>
-              Cancel
-            </Button>
-            <Button type="submit" form={manualAlertFormConfig.name} disabled={!formIsSubmittable}>
-              Create
-            </Button>
-          </HorizontalGroup>
-        </div>
+        <FormProvider {...formMethods}>
+          <form onSubmit={handleSubmit(onSubmit)} className={styles.form}>
+            <Controller
+              name="message"
+              control={control}
+              rules={{ required: 'Message is required' }}
+              render={({ field }) => (
+                <Field
+                  key="message"
+                  label="What is going on?"
+                  invalid={Boolean(errors.message)}
+                  error={errors.message?.message}
+                >
+                  <TextArea rows={4} {...field} />
+                </Field>
+              )}
+            />
+            <AddResponders mode="create" />
+            <div className="buttons">
+              <HorizontalGroup justify="flex-end">
+                <Button variant="secondary" onClick={onHideDrawer}>
+                  Cancel
+                </Button>
+                <Button type="submit" disabled={!formIsSubmittable}>
+                  Create
+                </Button>
+              </HorizontalGroup>
+            </div>
+          </form>
+        </FormProvider>
       </VerticalGroup>
     </Drawer>
   );
 });
+
+export const getStyles = () => ({
+  form: css`
+    width: 100%;
```

Comment:
there's a util style for `width: 100%`, it's called `width100` located in `utils.styles.ts`, no need to repeat classes with width: 100